### PR TITLE
add resources page for webinars and future content

### PR DESCRIPTION
**Pull request by hadar-co**
# Description

Add new Resources page for webinars and future content

## Added docs pages

Resources ('/resources')

### CODE REVIEW

File: docs/resources.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# 📚 Resources
+
+Here you will find more Port-related content that can help you get the most out of your developer portal.
+
+Feel free to reach out with any questions or requests 🚀
+
+- [Webinars](https://www.getport.io/webinars)
+
+- [Blog posts](https://www.getport.io/blog)
+
```
Comment (MPTG94) [inline docs/resources.md:16]: Let's also add the resource center - https://www.getport.io/resource-center
And the news room - https://www.getport.io/newsroom

File: docs/resources.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
```
Comment (MPTG94) [inline docs/resources.md:17]: We usually don't have a bold line at the end of a docs page, so I'm not sure about adding specifically here

**Review by MPTG94 — APPROVED** [review]
Looks good, refer to comments and merge

File: docs/resources.md (hunk 2 above)
Comment (hadar-co) [reply docs/resources.md:17]: @MPTG94  I think a visual separation from the "Edit this page" link is helpful. We can consider adding this in other pages

**Pull request merged**